### DISCUSSION TITLE
shift + enter autosuggest history not working

### DISCUSSION THREAD

**Discussion by Overcraft90** [ohmyzsh/ohmyzsh#12189] [Help] [answered]
Hi there, I've been using `oh-my-zsh` happily since a while now; however, for a stupid mistake I happened to delete the _.zshrc_ file which prompted me back with the old terminal style.

So, I deleted the folder for the tool, as the uninstall didn't work, the proceeded to reinstall it. I've been pretty much able to recover everything to to original state of things, but one issue presist.

Basically, before this happened I had an autosuggestion pane showing on the right of the command I was typing with some opition based on my history/previously commands used and items in the folder. At the moment, this thing is not working anymore... does anyone know how to get it back? Thanks in advance!

P. S. I'm using `iTerm2` whether this helps.

**Comment by carlosala** [comment, marked as answer]
It looks like you're missing https://github.com/zsh-users/zsh-autosuggestions.

**Reply by Overcraft90** [reply]
Hi @carlosala thanks for the quick answer; unfortunately, that is not the case. I already tried the `zsh-autosuggestions` but what it does is a simple shading of the previously used commands.

What I was talking about is a side pane appearing next to the command while writing, which in turn allows you to move with the UP/DOWN arrows to select the desired command, then with ENTER+SHIFT it prompts you with the desired command line to run. Let me know whether this helps, thanks again!

**Reply by rwmitchell** [reply]
Who marked that as the answer?  I assume only the OP would be able to do that but that doesn't match their follow-up comment.

While I don't use this particular feature, it sounds like something being done by iTerm2, not oh-my-zsh or any other separate plugin.  I would suggest looking thru the iTerm2 site for history window.
